storybookjs/storybook
# Switch to the only prepublish script

Issue: All packages use different versions of prepublish scripts. Plus some of them `shell` scripts which have bad compatibility with windows

## What I did

- Added the only instance of prepublish script to
`./scripts/prepublish.js`
it's the shell js version wich allready was used in [`storybook-ui`](https://github.com/storybooks/storybook-ui/blob/master/.scripts/prepublish.js)

- Changed in `scripts` of each `package.json` to
`"prepublish": "node ../../scripts/prepublish.js"`

- Removed previos versions of prepublish scripts from packages

## How to test

1. clone this repo
2. run `yarn`
it should install all dependencies without any errors and buid `dist`'s for all packages
